src/arcade/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoDefaults.props

```diff
@@ -11,15 +11,11 @@
     <PlatformName Condition="'$(PlatformName)' == ''">$(Platform)</PlatformName>
   </PropertyGroup>
 
-  <!-- DotNetBuild properties -->
   <PropertyGroup>
-    <!-- The DotNetBuild property is set to true if in any kind of .NET product build (orchestrator, repo, or inner repo). -->
-    <DotNetBuild Condition="'$(DotNetBuildInnerRepo)' == 'true' or '$(DotNetBuildRepo)' == 'true' or '$(DotNetBuildOrchestrator)' == 'true'">true</DotNetBuild>
-
-    <!-- The phase is calculated from the inner, orchestrator, and repo variables. Innermost wins. -->
-    <DotNetBuildPhase Condition="'$(DotNetBuildInnerRepo)' == 'true'">InnerRepo</DotNetBuildPhase>
-    <DotNetBuildPhase Condition="'$(DotNetBuildPhase)' == '' and '$(DotNetBuildRepo)' == 'true'">Repo</DotNetBuildPhase>
-    <DotNetBuildPhase Condition="'$(DotNetBuildPhase)' == '' and '$(DotNetBuildOrchestrator)' == 'true'">Orchestrator</DotNetBuildPhase>
+    <!-- The DotNetBuild property is set to true if in any kind of .NET product build (orchestrator or repo). -->
+    <DotNetBuild Condition="'$(DotNetBuildRepo)' == 'true' or '$(DotNetBuildOrchestrator)' == 'true'">true</DotNetBuild>
+    <!-- PrebuiltUsage is enabled by default when building source-only. -->
+    <PrebuiltUsage Condition="'$(PrebuiltUsage)' == '' and '$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetBuildRepo)' == 'true'">true</PrebuiltUsage>
```

Comment:
The name is misleading to me as it gives the impression prebuilts are used/detected. Referring to it as the `TrackPrebuilts` or `TrackPrebuiltUsage` feature is more descriptive IMO.

This would also help clarify some of the other `PrebuiltUage` prefixed properties like `PrebuiltUsagePackageCache`.  `PrebuiltUsagePackageCache` gives the impression to me that these are all of the prebuilts.

Reply:
Sure. Sounds like I should generally change the "PrebuiltUsage" property and prefix to something like "TrackPrebuiltUsage". Is that right?

Reply:
For the cache folder we currently have `.prebuilt-packages`. Should that also be changed?

I.e. `.prebuilt-package-cache` or `.track-prebuilts-cache`?

The intention here is to use a different cache than the default one set by Arcade scripts (`.packages`) to not accidentally build with a cache that already contains packages from the non-source-only build.

Reply:
> Sure. Sounds like I should generally change the "PrebuiltUsage" property and prefix to something like "TrackPrebuiltUsage". Is that right?

Yes, I think that would help clarify the purpose of the property(s).

Reply:
> For the cache folder we currently have .prebuilt-packages. Should that also be changed?

Yes I think it should.  What do you think about `.source-only-package-cache`?

Reply:
> Yes I think it should. What do you think about .source-only-package-cache?

I integrated the PrebuiltUsage feature in a way into the Arcade SDK that it is now usable for all kinds of builds. Even for non-source-only ones which could be useful to track dependencies. Based on that, it would be good to avoid `source-only` in the name.

Reply:
> I integrated the PrebuiltUsage feature in a way into the Arcade SDK that it is now usable for all kinds of builds.

I realize that.  It is just that the `.prebuilt-package-cache` or `.track-prebuilts-cache` names are difficult to understand from the perspective of someone building from source.  Would it be terrible to introduce a source-only specific name?

<NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' and '$(PrebuiltUsage)' == 'true' and '$(DotNetBuildSourceOnly)' == 'true'">$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', '.source-only-package-cache'))</NuGetPackageRoot>
<NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' and '$(PrebuiltUsage)' == 'true'">$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', '.track-prebuilts-cache'))</NuGetPackageRoot>

Reply:
Sure, that works.